app/views/batch_objects/show.html.erb

```diff
@@ -47,6 +47,32 @@
 	<%= content_tag :div, "No relationships in this batch object" %>
 <% end %>
 
+<%= content_tag :h4, "Attributes" %>
```

Comment:
I would prefer that we not use `content_tag` in templates like this.  That is, do this:

    <h4>Attributes</h4>

not

    <%= content_tag :h4, "Attributes" %>

Etc.

What do you think?